src/parser/jobs/rdm/modules/GeneralCDDowntime.ts

```diff
@@ -12,19 +12,19 @@ export default class GeneralCDDowntime extends CooldownDowntime {
 		// hope that it's accurate enough
 		{
```

Comment:
Should this comment be changed or renewed in light of the code change?

Reply:
We're not actually sure honestly, we tried a multitude of tests but weren't entirely happy with any result.  This particular configuration yielded the most correct data across different situations.  We elected in the TC channel to just go with it and see how it goes with more real world data.